completions/bash/oci-runtime-tool

```diff
@@ -363,28 +363,36 @@ _oci-runtime-tool_generate() {
 	local all_options="$options_with_args $boolean_options"
 
 	case "$prev" in
-		--seccomp-arch)
-		    __oci-runtime-tool_complete_seccomp_arches
+		--arch)
+		    __oci-runtime-tool_complete_arches
 			return
 			;;
 
-		--arch)
-		    __oci-runtime-tool_complete_arches
+		--cap-add|--cap-drop)
+			__oci-runtime-tool_complete_capabilities
 			return
 			;;
 
-		--seccomp-default)
-			__oci-runtime-tool_complete_seccomp_actions
+		--env|-e)
+			COMPREPLY=( $( compgen -e -- "$cur" ) )
+			__oci-runtime-tool_nospace
 			return
 			;;
 
-		--root-propagation)
-			__oci-runtime-tool_complete_propagations
+		--env-file)
+		    _filedir
+			__oci-runtime-tool_nospace
+		    return
+		    ;;
+
+		--gid)
+            _gids
```

Comment:
The _uids and _gids is same as _filedir(can be found at completions/bash/oci-runtime-tool:423). They all are library functions, and be placed in bash-completion.

You can find these functions in /usr/share/bash-completion/bash_completion